resources/templates/worker-config-ansible-playbook.yaml

```diff
@@ -154,12 +157,40 @@
         become: no
         ansible.builtin.stat:
           path: "{{ .OSBuildWorkerS3CABundleDir }}/{{ .OSBuildWorkerS3CABundleKey }}"
-        register: ca_bundle_file
+        register: s3_ca_bundle_file
       - name: Copy the S3 CA Bundle file if exists
         ansible.builtin.copy:
           src: "{{ .OSBuildWorkerS3CABundleDir }}/{{ .OSBuildWorkerS3CABundleKey }}"
           dest: {{"'{{ osbuild_worker_config_directory }}/{{ osbuild_worker_s3_ca_bundle_file }}'"}}
-        when: ca_bundle_file.stat.exists
+        when: s3_ca_bundle_file.stat.exists
+
+  - name: Handle the Container Registry Credential files
+    no_log: True
```

Comment:
Maybe need to set `no_log`to false?

Reply:
Why? Anything certificate related should not be reflected in the logs (e.g. where they are taken from and to where)